extern/default.nix

```diff
@@ -8,10 +11,15 @@
   overlays = [
     nur.overlay
     devshell.overlay
+    (final: prev: {
+      deploy-rs = deploy.packages.${prev.system}.deploy-rs;
+    })
   ];
 
   # passed to all nixos modules
   specialArgs = {
+    inherit hmModules;
+
```

Comment:
but how do I have a single comment
```
with code
```
blocks (does this edit show? Nope it duplicates)